experimental/framework/fluid-static/src/FluidStatic.ts

```diff
@@ -70,6 +75,12 @@ export class FluidContainer {
         return this.rootDataObject.createDataObject<T>(dataObjectClass, id);
     }
 
+    public getId() {
```

Comment:
why not just clientId?

Reply:
are we expecting the user to understand what the 'client' is? or do you think that term is generic enough?